api/placer.py

```diff
@@ -494,7 +495,8 @@ def finalize(self):
             'filename': self.name,
             'path':	 self.path,
             'size':	 os.path.getsize(self.path),
-            'hash':	 files.hash_file_formatted(self.path),
+            'hash': files.hash_file_formatted(self.path),
+            'uuid':	 str(uuid.uuid4()),
```

Comment:
How do you feel about naming this field `_id` and using it as the `_id` field in Mongo when files become their own collection? Any reasons why they should be separate?

Reply:
Agreed, let's follow Megan's convention if at all possible.